hash/msx1_cart.xml

```diff
@@ -24514,7 +24469,7 @@ legacy FM implementations cannot find it.
 	<software name="psged08" cloneof="psged">
 		<description>PSGed (v0.8b)</description>
 		<year>2010</year>
-		<publisher>&lt;homebrew&gt;</publisher>
+		<publisher>Aorante</publisher>
```

Comment:
This seems to always be stylised “aorante” (no initial capital) as in the `developer` info element.